ignite/handlers/lr_finder.py

```diff
@@ -493,9 +493,9 @@ def __init__(self, optimizer: Optimizer, start_lr: float, end_lr: float, num_ite
         super(_ExponentialLR, self).__init__(optimizer, last_epoch)
 
         # override base_lrs
-        self.base_lrs = start_lr
+        self.base_lrs = start_lr  # type: ignore
```

Comment:
As you already mentioned that in discord, let's try to explore why there is an error.

Reply:
Actually, the issue is that `self.base_lrs` is `List[float]` but `start_lr` is `float`. There is a type mismatch. I wonder if actually this `_ExponentialLR` is working correctly ...

Reply:
Oh, there is just a typehint to fix:

https://github.com/pytorch/ignite/blob/9402abeb67b35c2272ede7af4a2c64dd64aa7278/ignite/handlers/lr_finder.py#L125-L126

We can first fix typehint for this line:
```
start_lr = [start_lr] * len(optimizer.param_groups)  # type: ignore
```
and then update 
```
class _ExponentialLR(_LRScheduler):
    def __init__(self, optimizer: Optimizer, start_lr: float, end_lr: float, num_iter: int, last_epoch: int = -1):
```

Reply:
Yes. Is the intended use what the description says?

Reply:
Intended usage is `start_lr: List[float]`